build.sbt

```diff
@@ -186,6 +193,53 @@ def crossSparkSettings(): Seq[Setting[_]] = getSparkVersion() match {
   )
 }
 
+lazy val connectCommon = (project in file("spark-connect/common"))
+  .settings(
+    name := "delta-connect-common",
```

Comment:
why is there a common? seems like only connectServer is using it. so why have a common at all?

Reply:
This `common` holds the `Protobuf` messaging and utils functions that both the server and `future` Scala client will use.